service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -34,7 +34,7 @@ public Method getMethod(UUID methodId) {
   }
 
   public List<Method> getMethods() {
-    String sql = "SELECT * FROM method";
+    String sql = "SELECT * FROM method GROUP BY method.method_id ORDER BY MIN(created) DESC";
```

Comment:
It feels really surprising that you need the group by here... are you sure this doesn't work?
```suggestion
    String sql = "SELECT * FROM method ORDER BY created DESC";
```

Reply:
This totally works. I don't know why I thought I needed to use `MIN()`.  That was forcing me to use the `GROUP BY` method.